Modules/Server/Sources/PocketCastsServer/Public/Search/PodcastSearchTask.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+public struct PodcastSearchResult: Decodable {
+    public let uuid: String
+    public let title: String
+    public let author: String
+}
+
+public class PodcastSearchTask {
+    var session = URLSession.shared
+
+    public init() {}
+
```

Comment:
Same suggestion about injecting the session. 

```suggestion
    private let session: URLSession

    public init(session: URLSession = .shared) {
        self.session = session
    }
```